web/docs/tql2/operators/to_elasticsearch.md

```diff
@@ -0,0 +1,117 @@
+# to_elasticsearch
+
+Sends events to an ElasticSearch-compatible Bulk API.
+
+```tql
+to_elasticsearch url:string, action=string, [index=string, id=string, doc=record,
+    user=string, passwd=string, skip_peer_verification=bool, cacert=string,
+    certfile=string, keyfile=string, include_nulls=bool, max_content_length=int,
+    buffer_timeout=duration, compress=bool]
+```
+
+## Description
+
+The `to_elasticsearch` operator sends events to a [ElasticSearch-compatible Bulk
+API][api].
+
+[api]: https://opensearch.org/docs/latest/api-reference/document-apis/bulk/
+
+The operator accumulates multiple events before sending them as a single
+request. You can control the maximum request size via the
+`max_content_length` and the timeout before sending all accumulated events via
+the `send_timeout` option.
+
+### `url: string`
+
+The URL of the API endpoint.
+
+### `action = string`
+
+An expression for the action that evaluates to a `string`.
+
+Supported actions:
+- `create`: Creates a document if it doesn’t already exist and returns an error otherwise.
+- `delete`: Deletes a document if it exists.
+- `index`: Creates a document if it doesn’t yet exist and replace the document if it already exists.
+- `update`: Updates existing documents and returns an error if the document doesn’t exist.
+- `upsert`: If a document exists, it is updated; if it does not exist, a new document is indexed.
+
+### `index = expr (optional)`
```

Comment:
```suggestion
### `index = string (optional)`
```

I know we have done this differently in other places, but I feel like this should probably say `string`. If anything, this shows that we need a better way to document the different requirements for runtime expressions and compile time constants.

I would be fine with using either `string` or `expr` here, but we should be consistent *and* be consistent with the synopsis.